client/src/components/layouts/default.tsx

```diff
@@ -26,46 +25,75 @@ import Flash from '../Flash';
 import Header from '../Header';
 import Footer from '../Footer';
 // preload common fonts
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
```

Comment:
Hey there. Would you mind changing these to `@ts-expect-error` - that way, you can also remove the `ban-ts-comment` comment.